src/cli.js

```diff
@@ -62,12 +63,23 @@ export async function signInToGithub() {
   return credentials;
 }
 
+/**
+ * Facilitates prompting the user for their username and password
+ * @returns credentials the username and password of the user
+ */
+export async function signInToSequelize() {
```

Comment:
change the name of this function to signIntoMysql in case other dbs in the future need mysql credentials

Reply:
Not going to change to keep since this just calls a common method anyways and we might want to add new tools later, which would require a separate function.